plugin-directorymenu/directorymenu.cpp

```diff
@@ -102,6 +105,12 @@ void DirectoryMenu::openDirectory(const QString& path)
     QDesktopServices::openUrl(QUrl("file://" + QDir::toNativeSeparators(path)));
 }
 
+void DirectoryMenu::openInTerminal(const QString& path)
+{
+    QString action = mDefaultTerminal + " --workdir " + path;
```

Comment:
With this you (or the attacker) can trick the user to execute (malicious) code, right? Please, do not assume that `path` is a nice-looking-no-space-or-other-evil string.